src/screens/TopicMemberScreen/index.js

```diff
@@ -1,13 +1,6 @@
 import * as React from 'react';
-import {
-  Animated,
-  InteractionManager,
-  Keyboard,
-  ScrollView,
-  StyleSheet,
-  Platform
-} from 'react-native';
-import {useNavigation, useRoute} from '@react-navigation/native';
+import {Animated, ScrollView, StyleSheet, Platform} from 'react-native';
+import {useRoute} from '@react-navigation/native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `InteractionManager` and `Keyboard` from the import statement is a good change if these modules are not being used in the file. This helps to keep the code clean and maintainable.
```diff
- import {Animated, InteractionManager, Keyboard, ScrollView, StyleSheet, Platform} from 'react-native';
+ import {Animated, ScrollView, StyleSheet, Platform} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->